### PR TITLE
Centres placeholder text in Safari

### PR DESCRIPTION
**Pull request by ashfurrow**
This PR is for [PURCHASE-461](https://artsyproduct.atlassian.net/browse/PURCHASE-461) to centre the placeholder text in Safari (and thus, in Eigen as well). It removes some unused CSS as well, and I've opened [a Jira ticket](
https://artsyproduct.atlassian.net/browse/PURCHASE-518) to further investigate simplifying the input/mixins code.

screen shot 2018-10-02 at 12 36 01 pm

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 681d2344da6fd4a6adf4d1eb5a0d279be7f81b7a

https://deploy-preview-1347--artsy-reaction.netlify.com

**Comment by ashfurrow** [comment]
Hmm, snapshot updates seem noisy. Let's see what CI says.

File: src/Components/Mixins.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const borderedInput = (props: InputProps & BorderProps = {}) => {
     &::placeholder {
       color: ${colors.grayMedium};
       text-overflow: ellipsis;
+      line-height: normal;
```
Comment (zephraph) [inline src/Components/Mixins.tsx]: 👍

**Comment by zephraph** [comment]
I'm not exactly sure what's up with the snapshots here, but we'll need to resolve these failures before continuing. I swear I was seeing this earlier today though. Hmm...

**Comment by ashfurrow** [comment]
Weird, okay!

**Comment by ashfurrow** [comment]
I pulled from upstream master, `rm -rf node_modules`, and it worked!

**Pull request merged**